recipes/pyflow/meta.yaml

```
@@ -0,0 +1,47 @@
+{% set name = "pyflow" %}
+{% set version = "2.0.1" %}
+
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://github.com/wonderworks-software/PyFlow/archive/{{ version }}.tar.gz"
+  sha256: "9a4a481ed9ccfa5295c87bdb32e7c26036351d2857bb9b2e9997c22fc681b856"
```

Could you please remove the quotes from ``sha256``?
```suggestion
  sha256: 9a4a481ed9ccfa5295c87bdb32e7c26036351d2857bb9b2e9997c22fc681b856
```